narwhal/consensus/src/consensus.rs

```diff
@@ -38,6 +38,11 @@ pub struct ConsensusState {
     pub last_committed: HashMap<PublicKey, Round>,
     /// Used to populate the index in the sub-dag construction.
     pub latest_sub_dag_index: SequenceNumber,
+    /// The last calculated consensus reputation score
+    pub last_consensus_reputation_score: ReputationScores,
```

Comment:
It may be more robust in future to group these fields related to last committed leader into a single struct, to ensure they are updated together.

Reply:
Agreed, I could refactor this on upcoming PR